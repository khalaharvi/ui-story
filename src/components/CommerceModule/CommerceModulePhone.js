import React from 'react';
import PropTypes from 'prop-types';

import { cx } from './CommerceModule';

const CommerceModulePhone = ({ name, phone }) => (
  <p className={cx('phone')}>Or call {name}: <a href={`tel:${phone}`}>{phone}</a></p>
);

CommerceModulePhone.propTypes = {
  phone: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired
};

export default CommerceModulePhone;
