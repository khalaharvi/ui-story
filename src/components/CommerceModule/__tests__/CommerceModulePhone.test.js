import React from 'react';
import { shallow } from 'enzyme';

import CommerceModulePhone from '../CommerceModulePhone';

describe('<CommerceModulePhone />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <CommerceModulePhone
        name="Awesome Showroom"
        phone="+33123456789"
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
