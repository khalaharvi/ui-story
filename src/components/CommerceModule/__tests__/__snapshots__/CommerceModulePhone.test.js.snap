// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommerceModulePhone /> renders correctly 1`] = `
<p
  className="dh-commerce-module-phone"
>
  Or call 
  Awesome Showroom
  : 
  <a
    href="tel:+33123456789"
  >
    +33123456789
  </a>
</p>
`;
